app/main.py

```diff
@@ -1,2 +1,59 @@
+from typing import Hashable, Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.threshold = int(self.capacity * 2 / 3)
+        self.buckets = [[] for _ in range(self.capacity)]
+        self.size = 0
+
+    def __getitem__(self, key: Hashable) -> Any:
+        hash_item = hash(key)
+        idx = hash_item % self.capacity
+
+        while self.buckets[idx]:
+            if (self.buckets[idx][1] == key
```

Comment:
If keys are equal, hash_items can't be different. When first part is True, second part always will be True either.
So there is no need to use 'and' part